app/src/main/java/com/example/cocktails/ui/main/ui/UserItem/SelectedCocktailImgLevel1.kt

```diff
@@ -0,0 +1,73 @@
+package com.example.cocktails.ui.main.ui.UserItem
+
+import android.content.Intent
+import android.net.Uri
+import android.os.Bundle
+import android.widget.Toast
+import androidx.appcompat.app.AppCompatActivity
+import androidx.recyclerview.widget.GridLayoutManager
+import androidx.recyclerview.widget.LinearLayoutManager
+import androidx.recyclerview.widget.RecyclerView
+import com.example.cocktails.R
+import com.google.firebase.storage.FirebaseStorage
+import com.google.firebase.storage.StorageReference
+
+
+class SelectedCocktailImgLevel1 : AppCompatActivity(),ImageAdapter.ItemClickListener {
+    private lateinit var mRecyclerView: RecyclerView
+    private lateinit var mAdapter: ImageAdapter
+    private lateinit var mIcons:ArrayList<IconItem>
+    private lateinit var mDataRef: StorageReference
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_selcect_cocktail_img_level1)
+
+        mRecyclerView = findViewById(R.id.recyclerView)
+        mRecyclerView.setHasFixedSize(true)
+        mRecyclerView.layoutManager = LinearLayoutManager(this)
+        mRecyclerView.layoutManager=GridLayoutManager(this, 3)
+        mIcons = ArrayList()
+
+        mDataRef = FirebaseStorage.getInstance().reference
```

Comment:
@enavhasson already exist under 'cocktails' class (use applicationContext.mStorageRef instead)

Reply:
I know . I checked for a bug . I say that this is not my final code.